tests/test_dataapi.py

```diff
@@ -0,0 +1,138 @@
+import unittest
+import responses
+from learnosity_sdk.request import DataApi
+
+# This test uses the consumer key and secret for the demos consumer
+# this is the only consumer with publically available keys
+# Normally the consumer secret should not be committed to a public
+# repository like this one.
```

Comment:
Could we move those comments straight next to the sensitive keys? We don't want people to skim over them (: